packages/components/stories/MultiSelect.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
+import PropTypes from 'prop-types';
 
 import MultiSelect from '../src/MultiSelect';
 import IconsProvider from '../src/IconsProvider';
 
-function onSelect(value, title) {
-	return {
-		onSelect: action(`selec ${value} ${title}`),
+/**
+ * ControlledMultiSelect reproduce the ComponentForm behaviors
+ */
+function ControlledMultiSelect(props) {
```

Comment:
I don't get this component. It has the same behavior than the uncontrolled one.
To demonstrate the controlled component, the state should not move, and the actions should be triggered as we interact.
This behavior should be demonstrated in the component form story